webapp/components/features/FollowList/FollowList.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <base-card class="hc-connections">
+    <ds-space v-if="this.connections && this.connections.length" margin="x-small">
```

Comment:
```suggestion
    <ds-space v-if="connections && connections.length" margin="x-small">
```
don't know how this isn't throwing an error, but you don't have access to this in the `html`